index.js

```diff
@@ -97,26 +122,31 @@ function myMap(array, callback) {
   return result
 }
 
+function triple(element) {
+  return element * 3
+}
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
+console.log(myMap([1,2,3,4,5], triple)) 
 // // => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
+console.log(myMap([2,4,6,8], triple))   
 // // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
-
+console.log("%c----------", "color: red") 
 
+function greet(element) {
+  return element = `Hi ${element}`
+}
```

Comment:
This is correct! you don't have to return it with the `element =` though, you can just return the string!